client/src/app/app.module.ts

```diff
@@ -1,74 +1,74 @@
-import { MatIconModule } from '@angular/material/icon';
-import { MatListModule } from '@angular/material/list';
+import { MatCardModule } from '@angular/material/card';
+import { MatDividerModule } from '@angular/material/divider';
+import { MatExpansionModule } from '@angular/material/expansion';
 import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatIconModule } from '@angular/material/icon';
 import { MatInputModule } from '@angular/material/input';
-import { MatCardModule } from '@angular/material/card';
+import { MatListModule } from '@angular/material/list';
 import { MatMenuModule } from '@angular/material/menu';
-import { MatExpansionModule } from '@angular/material/expansion';
-import { MatTooltipModule } from '@angular/material/tooltip';
-import { MatSelectModule } from '@angular/material/select';
 import { MatOptionModule } from '@angular/material/core';
-import { MatDividerModule } from '@angular/material/divider';
 import { MatRadioModule } from '@angular/material/radio';
+import { MatSelectModule } from '@angular/material/select';
+import { MatSidenavModule } from '@angular/material/sidenav';
 import { MatSnackBarModule } from '@angular/material/snack-bar';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatTooltipModule } from '@angular/material/tooltip';
 
-import { UserListComponent } from './users/user-list.component';
+import { AddUserComponent } from './users/add-user.component';
+import { AppComponent } from './app.component';
+import { AppRoutingModule } from './app-routing.module';
 import { HomeComponent } from './home/home.component';
-import { UserService } from './users/user.service';
-import { HttpClientModule } from '@angular/common/http';
-import { LayoutModule } from '@angular/cdk/layout';
 import { UserCardComponent } from './users/user-card.component';
+import { UserListComponent } from './users/user-list.component';
 import { UserProfileComponent } from './users/user-profile.component';
-import { AddUserComponent } from './users/add-user.component';
+import { UserService } from './users/user.service';
 
 const MATERIAL_MODULES: any[] = [
-  MatListModule,
   MatButtonModule,
-  MatIconModule,
-  MatToolbarModule,
   MatCardModule,
-  MatMenuModule,
-  MatSidenavModule,
-  MatInputModule,
+  MatDividerModule,
   MatExpansionModule,
-  MatTooltipModule,
-  MatSelectModule,
-  MatOptionModule,
   MatFormFieldModule,
-  MatDividerModule,
+  MatIconModule,
+  MatInputModule,
+  MatListModule,
+  MatMenuModule,
+  MatOptionModule,
   MatRadioModule,
-  MatSnackBarModule
+  MatSidenavModule,
+  MatSelectModule,
+  MatSnackBarModule,
+  MatToolbarModule,
+  MatTooltipModule,
 ];
 
 @NgModule({
   declarations: [
+    AddUserComponent,
     AppComponent,
     HomeComponent,
-    UserListComponent,
     UserCardComponent,
+    UserListComponent,
     UserProfileComponent,
-    AddUserComponent,
   ],
   imports: [
```

Comment:
The order of these matters. I would be careful arbitrarily reordering them like this. If two modules have providers for the same token, the later one wins. Also routing order is affected by the order of routing modules here.